src/raftstore/store/peer.rs

```diff
@@ -401,7 +425,8 @@ impl Peer {
         if req.has_admin_request() {
             match req.get_admin_request().get_cmd_type() {
                 AdminCmdType::CompactLog |
-                AdminCmdType::InvalidAdmin => {}
+                AdminCmdType::InvalidAdmin |
+                AdminCmdType::TransferLeader => {}
```

Comment:
Why transfer leader is same as split? 

Don't suppose that we can't receive any stale TransferLeader command.